terraform/aws/instances.tf

```diff
@@ -454,6 +454,80 @@ resource "aws_instance" "logs" {
     DashNetwork = terraform.workspace
   }
 
+  lifecycle {
+    ignore_changes = [ami]
```

Comment:
I guess we should remove it

Reply:
Oh yeah looks like a copy paste error on my part...

Reply:
Wait... why are we removing this? It's attached to the logs section, we haven't touched it in this PR?